version/version.go

```diff
@@ -0,0 +1,5 @@
+package version
+
+var (
+	Version = "0.0.1"
```

Comment:
out of curiosity are we going to manually update this file or will the versions be updated via software with each green build on master?

Reply:
Yes this will have to manually updated for the time being. Once we start cutting releases we can address this issue.